diff --git a/node_modules/@nuxt/content-theme-docs/src/components/app/AppSearch.vue b/node_modules/@nuxt/content-theme-docs/src/components/app/AppSearch.vue
index 31ce990..6028a9e 100644
--- a/node_modules/@nuxt/content-theme-docs/src/components/app/AppSearch.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/components/app/AppSearch.vue
@@ -31,7 +31,7 @@
       :class="{ 'rounded-t-none': focus && (searching || results.length) }"
       style="margin-top: 37px;"
     >
-      <li v-if="searching && !results.length" class="px-4 py-2">Searching...</li>
+      <li v-if="searching && !results.length" class="px-4 py-2">查詢中...</li>
       <li
         v-for="(result, index) of results"
         :key="result.slug"
diff --git a/node_modules/@nuxt/content-theme-docs/src/components/app/AppSubtitle.vue b/node_modules/@nuxt/content-theme-docs/src/components/app/AppSubtitle.vue
new file mode 100644
index 0000000..f92baf2
--- /dev/null
+++ b/node_modules/@nuxt/content-theme-docs/src/components/app/AppSubtitle.vue
@@ -0,0 +1,18 @@
+<template>
+  <div>
+    <div v-if="document.subtitle">
+      <p class="text-gray-600 dark:text-gray-400">{{ document.subtitle }}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    document: {
+      type: Object,
+      required: true,
+    },
+  },
+};
+</script>
diff --git a/node_modules/@nuxt/content-theme-docs/src/i18n/zh-TW.js b/node_modules/@nuxt/content-theme-docs/src/i18n/zh-TW.js
new file mode 100644
index 0000000..1ee87e6
--- /dev/null
+++ b/node_modules/@nuxt/content-theme-docs/src/i18n/zh-TW.js
@@ -0,0 +1,12 @@
+export default {
+    search: {
+        placeholder: '搜尋文件(按"/" 查詢)'
+    },
+    toc: {
+        title: '文件目錄'
+    },
+    article: {
+        github: '在GitHub上在編輯此頁面',
+        updatedAt: '更新時間：'
+    }
+}
diff --git a/node_modules/@nuxt/content-theme-docs/src/pages/_.vue b/node_modules/@nuxt/content-theme-docs/src/pages/_.vue
index d67127b..749cf88 100644
--- a/node_modules/@nuxt/content-theme-docs/src/pages/_.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/pages/_.vue
@@ -17,9 +17,7 @@
           {{ document.title }}
           <Badge v-if="document.badge">{{ document.badge }}</Badge>
         </h1>         
-        <div v-if="document.subtitle" class="-mt-4">
-          <p class="text-gray-600 dark:text-gray-400">{{ document.subtitle }}</p>
-        </div>
+        <AppSubtitle :document="document" class="-mt-4" />
 
         <NuxtContent :document="document" />
       </article>
diff --git a/node_modules/@nuxt/content-theme-docs/src/tailwind.config.js b/node_modules/@nuxt/content-theme-docs/src/tailwind.config.js
index 2e83ca9..a981c0b 100644
--- a/node_modules/@nuxt/content-theme-docs/src/tailwind.config.js
+++ b/node_modules/@nuxt/content-theme-docs/src/tailwind.config.js
@@ -209,7 +209,26 @@ module.exports = ({ nuxt }) => ({
         zIndex: false,
         opacity: false
       }
-    )
+    ),
+    function ({ addComponents }) {
+      addComponents({
+        '.container': {
+          maxWidth: '100%',
+          '@screen sm': {
+            maxWidth: '640px',
+          },
+          '@screen md': {
+            maxWidth: '768px',
+          },
+          '@screen lg': {
+            maxWidth: '1280px',
+          },
+          '@screen xl': {
+            maxWidth: '1536px',
+          },
+        }
+      })
+    }    
   ],
   purge: {
     // Learn more on https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
